.github/workflows/nightly.yml

```diff
@@ -0,0 +1,118 @@
+name: Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+
+
+permissions:
+  contents: write
+  packages: write
+  issues: write
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.22.0
```

Comment:
Can we use `go-version: stable` here too?

Reply:
I vaguely remember that I need to pin to 1.22 (I think it might have just been for something Android related), but it does support the stable alias.

https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases